src/locales/en-US/recommender.ts

```diff
@@ -0,0 +1,17 @@
+export default {
+  'recommender.title': 'Recommender Wellms',
+  'recommender.add_exercise': 'Add an exercise now ',
+  'recommender.increase': 'to increase the course pass rate ',
+  'recommender.add_button': 'Yes, add',
+  'recommender.no_recommendation': 'No recommendations',
+  'recommender.finish': 'Finish this lesson now. ',
+  'recommender.too_long': 'Lessons that are too long cause a drop in pass rates ',
+  'recommender_info.title': 'How do we know this?',
+  'recommender_info.first_title': 'Recommender for better Educational results.',
+  'recommender_info.first_text':
+    'Wellms continually strives to improve the experience and ease of adding multimedia content. How can we be sure that subsequent exercise and lesson recommendations are worth adding? The answer lies in advanced artificial intelligence algorithms and precise calculations, which are an integral part of our system based on XGBoost, i.e. eXtreme Gradient Boosting, algorithms.',
+  'recommender_info.second_title': 'The recommender analyzes the content:',
+  'recommender_info.second_text':
+    'The Wellms Recommender analyzes the content of each exercise and lesson for complexity, comprehensibility, and consistency with other course materials. Recommending next steps or completion percentage. Machine learning algorithms analyze patterns in the data, helping to identify exercises and lessons that best suit students individual preferences and skill levels.',
+  'recommender.add_new_exercise': 'Add exercise',
+};
```

Comment:
Wouldn't it be better?
```suggestion
export default {
  recommender: {
    title: "Recommender Wellms",
    add_exercise: "Add an exercise now ",
    increase: "to increase the course pass rate ",
    add_button: "Yes, add",
    no_recommendation: "No recommendations",
    finish: "Finish this lesson now. ",
    too_long: "Lessons that are too long cause a drop in pass rates ",
    add_new_exercise: "Add exercise",
    info: {
      title: "How do we know this?",
      first_title: "Recommender for better Educational results.",
      first_text:
        "Wellms continually strives to improve the experience and ease of adding multimedia content. How can we be sure that subsequent exercise and lesson recommendations are worth adding? The answer lies in advanced artificial intelligence algorithms and precise calculations, which are an integral part of our system based on XGBoost, i.e. eXtreme Gradient Boosting, algorithms.",
      second_title: "The recommender analyzes the content:",
      second_text:
        "The Wellms Recommender analyzes the content of each exercise and lesson for complexity, comprehensibility, and consistency with other course materials. Recommending next steps or completion percentage. Machine learning algorithms analyze patterns in the data, helping to identify exercises and lessons that best suit students individual preferences and skill levels.",
    },
  },
};
```

Reply:
We do it this way everywhere:
![Zrzut ekranu 2023-11-14 o 07 41 30](https://github.com/EscolaLMS/Admin/assets/62983152/0db8b03d-3bd5-43df-bc42-47bea374f80c)